After https://github.com/boostorg/logic/commit/23cd89d4c80f build fails:

sfx2/source/appl/shutdownicon.cxx:170:12: error: no viable conversion from returned value of type 'boost::logic::tribool' to function return type 'bool'
    return loaded;
           ^~~~~~

--- sfx2/source/appl/shutdownicon.cxx.orig	2019-02-05 07:22:07.809681000 +0800
+++ sfx2/source/appl/shutdownicon.cxx	2019-02-05 07:21:48.200022000 +0800
@@ -143,7 +143,7 @@ bool LoadModule()
 #endif // ENABLE_QUICKSTART_APPLET
     }
     assert(!boost::logic::indeterminate(loaded));
-    return loaded;
+    return bool{loaded};
 }
 
 }
